doc(pe): Document policy options (#601)
armory/docs: content/en/docs/plugin-guide/plugin-policy-engine/packages/spinnaker.execution/pipelines.before.md

| `input.pipeline.keepWaitingPipelines` | `boolean` | If false and concurrent pipeline execution is disabled, then the pipelines in the waiting queue will get canceled when the next execution starts. |
| `input.pipeline.limitConcurrent`                   | `boolean`  | Disable concurrent pipeline executions (only run one at a time).                                                                                                     |
| `input.pipeline.name`                              | `string`   | The name of the pipeline.                                                                                                                                            |
| `input.pipeline.parallel`                          | `boolean`  |                                                                                                                                                                      |
| `input.pipeline.parameterConfig[].default`         | `string`   | The default value associated with this parameter.                                                                                                                    |

> | `input.pipeline.parameterConfig[].description`     | `string`   | if supplied, will be displayed to users as a tooltip when triggering the pipeline manually. You can include HTML in this field.                                      |

Reviewer: ```suggestion
| `input.pipeline.parameterConfig[].description`     | `string`   | If supplied, the description will be displayed to users as a tooltip when triggering the pipeline manually. You can include HTML in this field.                                      |
```

| `input.pipeline.parameterConfig[].hasOptions`      | `boolean`  | True if the **Show Options** checkbox in the parameter is checked.                                                                                                   |

> | `input.pipeline.parameterConfig[].label`           | `string`   | A label to display when users are triggering the pipeline manually.                                                                                                  |

Reviewer: ```suggestion
| `input.pipeline.parameterConfig[].label`           | `string`   | A label to display when users trigger the pipeline manually.                                                                                                  |
```

| `input.pipeline.parameterConfig[].pinned`          | `boolean`  | If checked, this parameter will be always shown in a pipeline execution view, otherwise it'll be collapsed by default.                                               |
| `input.pipeline.parameterConfig[].required`        | `boolean`  | If true, the user must provide a value for this parameter when executing the pipeline.                                                                               |
| `input.pipeline.plan`                              | `boolean`  |                                                                                                                                                                      |
| `input.pipeline.respectQuietPeriod`                | `boolean`  |                                                                                                                                                                      |
| `input.pipeline.spelEvaluator`                     | `string`   | Which version of SpEL should SpEL expressions be evaluated with.                                                                                                     |

> | `input.pipeline.stages[]`                          | `[array]`  | An array of the stages in the pipeline. Typically if you are writing a policy that examines multiple pipeline stages, it is better to write that policy against either the `opa.pipelines package`, or the `spinnaker.execution.pipelines.before` package. |

Reviewer: ```suggestion
| `input.pipeline.stages[]`                          | `[array]`  | An array of the stages in the pipeline. Typically if you are writing a policy that examines multiple pipeline stages, it is better to write that policy against either the `opa.pipelines package` or the `spinnaker.execution.pipelines.before` package. |
```
